src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderMovementResourceApiTest_moveToCell.java

```diff
@@ -101,14 +101,6 @@ public void bookingId_notFound() {
         assertThat(getBodyAsJsonContent(response)).extractingJsonPathStringValue("$.userMessage").isEqualTo(bookingNotFoundError);
     }
 
-    @Test
-    public void bookingId_missing_notFound() {
-        final var response = testRestTemplate.exchange("/api/bookings//living-unit/LEI-A-1-1?reasonCode=ADM&dateTime=2020-03-24T13:24:35", PUT, anEntity(), String.class);
```

Comment:
this was then defaulting to the global error handler so not actually calling this endpoint.  Pointless test.